### PR TITLE
Add type definition for HLRU

### PR DESCRIPTION
**Pull request by alxolr**
Added type definition, for typescript users

### CODE REVIEW

**Comment by davidmarkclements** [comment]
Hey thanks for the PR

Why would type definitions be a minor and not a patch?

In other projects where we've included typescript definitions I've found that maintenance burden is increased purely because typescript definition format has changed over time. In one project (fast-safe-stringify) I took on that burden. In another  (pino) we asked for typescript definitions to be managed separately by the typescript community under the @types project. 

I'll leave it to @dominictarr to weigh in on this

**Comment by alxolr** [comment]
About patch or minor was not sure, usually when it is a new feature then is a minor, patches are for bug-fixes, so at least this is how I understood semver. It is up to you, to decide... Concerning @types this also makes sense but my problem with @types that they stay behind, and a lot of the times you may have misleading types.

In my opinion definitions and the code should be on the same repository, cause they will be up to date most of the time, is much easier when you write everything in typescript, the definitions are automatically generated.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "hashlru",
   "description": "simpler faster substitute for LRU",
-  "version": "2.2.1",
+  "version": "2.3.1",
```
Comment (dominictarr) [inline package.json:4]: please leave the maintainer to set the version number when they merge, don't set it in the PR.

**Comment by dominictarr** [comment]
merged into 2.3.0

**Pull request merged**